cves/CVE-2007-1862.yml

```diff
@@ -31,7 +31,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: When Apache would copy header data for a response header, it
```

Comment:
Good start. Explain what a header is. Where would this land to a user? What would be the consequence.

Just dig deeper.

Reply:
I tried fixing this in a commit a few days ago, forgot to respond to the comment.